app/js/react.force.common.js

```diff
@@ -1,76 +0,0 @@
-/*
```

Comment:
The react.force.*.js are copied from the external/shared folder when install.sh is run, so they don't need to be checked in